src/pages/code-reviews-getting-started/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Code reviews: getting started"
+date: 2018-05-03T12:07:50+03:00
+draft: false
+---
+
+What if I told you there is this really straightforward technique to make your project run smoother, faster, and with fewer bugs 🤔. Better yet, it’s completely free, requires only minimal tooling, and any developer can learn to use it in a matter of minutes 😳. This is how good code reviews are, and why you should use them.
+
+_Author: Thibaud Colas, Software Architect at Codemate Ltd._
+
+---
+
+## Why it’s good
+
+Developers tend to think of productivity in terms of “how fast can I write the code for this new feature”. We also tend to say “I write code” when describing our jobs. In practice though, professional software development is a matter of understanding the code that is already there: we simply spend much more time reading than writing. Productivity thus depends more on the overall code’s quality and maintainability, rather than the skills of individual contributors making changes.
+
+Joel Spolsky puts it neatly:
+
+> [It’s harder to read code than to write it](https://www.joelonsoftware.com/2000/04/06/things-you-should-never-do-part-i/).
+
+John Papa recently gave a great talk about what code readability means in practice:
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/56mETnrByBM" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+If you like numbers, another great article on the topic is [Code Reviews: Just Do It](https://blog.codinghorror.com/code-reviews-just-do-it/) by Jeff Atwood. You should definitely read it. I especially like its excerpts from [Code Complete](https://www.amazon.com/exec/obidos/ASIN/0735619670):
```

Comment:
I feel like "you should definitely read it" doesn't really tell the reader _why_ they should read it or why is it a valuable read.

Suggestions:
* tell the reader _why_ the article is such a good read 
* AND/OR drop the "You should definitely read it" statement altogether

Reply:
The reason is "If you like numbers", implying that the article contains statistics. If that's not clear enough I should probably reword, yup.

Reply:
In that case, maybe:

> Another great article on the topic is [Code Reviews: Just Do It](https://blog.codinghorror.com/code-reviews-just-do-it/) by Jeff Atwood. If you like numbers, you should definitely read it.

Reply:
👌 edited.